docs/timeseries/analysis.rst

```diff
@@ -0,0 +1,213 @@
+.. _timeseries-analysis:
+
+Manipulation and analysis of time series
+****************************************
+
+.. |Table| replace:: :class:`~astropy.table.Table`
+.. |TimeSeries| replace:: :class:`~astropy.timeseries.TimeSeries`
+.. |BinnedTimeSeries| replace:: :class:`~astropy.timeseries.BinnedTimeSeries`
+
+Combining time series
+=====================
+
+The  :func:`~astropy.table.vstack`, and :func:`~astropy.table.hstack` functions
+from the :mod:`astropy.table` module can be used to stack time series in
+different ways.
+
+Time series can be stacked 'vertically' or row-wise using the
+:func:`~astropy.table.vstack` function (although note that sampled time
+series cannot be combined with binned time series and vice-versa)::
+
+    >>> from astropy.table import vstack
+    >>> from astropy import units as u
+    >>> from astropy.timeseries import TimeSeries
+    >>> ts_a = TimeSeries(time='2016-03-22T12:30:31',
+    ...                          time_delta=3 * u.s,
+    ...                          data={'flux': [1, 4, 5, 3, 2] * u.mJy})
+    >>> ts_b = TimeSeries(time='2016-03-22T12:50:31',
+    ...                          time_delta=3 * u.s,
+    ...                          data={'flux': [4, 3, 1, 2, 3] * u.mJy})
+    >>> ts_ab = vstack([ts_a, ts_b])
+    >>> ts_ab
```

Comment:
Note: So, while indexing would sort by time, vstacking does not sort the result by time. Interesting...

I wonder if it is worth a mention of how to handle duplicates in vstack. You said that multiple `time` columns are not allowed (in the hstack section) because that would be confusing, but wouldn't duplicate time entries in the same column be equally confusing?

Reply:
Again this behavior is inherited from Table/QTable, not a decision here. We don't currently disallow duplicate time entries in the same column for now, as there might be use cases where that is valid (event lists?). I've added a note about the sorting/duplicates behavior.